lib/ansible/modules/windows/win_pester.py

```diff
@@ -26,6 +26,12 @@
       - If the path is a folder, the module will consider all ps1 files as Pester tests.
     type: str
     required: true
+  tags:
+    description:
+      - Runs only tests in Describe blocks with specified Tags values.
+      - Accepts multiple comma seperated tags.
+    type: str
```

Comment:
since it is 'tags' should this not be type 'list'?

Reply:
The internal logic is by taking a comma separated string and splitting it. Hence put it as string.

Reply:
You still want type `list`. The code is able to automatically split values by `,` or handle a literal list being passed in from the playbook. You would just need to make sure you code expects the value to be a list in the first place.

Reply:
Cool, @jborean thanks for that info, I didn't know that. Changed to `list`.